tests/integrations/test_dbm.py

```diff
@@ -55,7 +55,7 @@ def weblog_trace_payload(self):
                 ),
 
     def _get_db_span(self, response):
-        assert response.status_code == 200, f"Request: {self.scenario} wasn't successful."
+        assert response.status_code == 200, f"Request: {context.scenario.name} wasn't successful."
```

Comment:
you could use `self.scenario_name` instead. Not sure what is the best here

Reply:
the original one, there is not real added value to use `self` for other stuff than the one created only in setup method